src/components/card/cardWrapper/CardWrapper.tsx

```diff
@@ -0,0 +1,21 @@
+import * as React from "react";
+import * as styles from "./CardWrapper.module.css";
+import { HTMLAttributes } from "react";
+import clsx from "clsx";
+
+interface CardWrapperProps {
+  children: React.ReactNode;
+}
+
+export const CardWrapper = ({
+  children,
+  onClick,
+  id,
+  className,
```

Comment:
I think, adding these extra props to the wrapper is a good idea, especially because it needs to be a very generic component.

However, I think we've now cherry-picked a couple of props, which we _now_ think are relevant (onClick, id, className). I could imagine that in the future we would need more, requiring us to update this component.

It might be better to pass all possible props to the container div, this would also include the onClick et cetera. 


```
interface CardWrapperProps {
  children: React.ReactNode;
}

export const CardWrapper = (props: CardWrapperProps & React.HTMLAttributes<HTMLDivElement>): JSX.Element => {
  return (
    <div className={clsx(styles.container, props.className && props.className)} {...props}>
      {props.children}
    </div>
  );
};
```

And we can even simplify that further, because `:React.HTMLAttributes<HTMLDivElement>` also has `.children` in its props, removing the need for our own interface, so the file could become:

```
import * as React from "react";
import * as styles from "./CardWrapper.module.css";
import { HTMLAttributes } from "react";
import clsx from "clsx";

export const CardWrapper = (props: React.HTMLAttributes<HTMLDivElement>): JSX.Element => {
  return (
    <div className={clsx(styles.container, props.className && props.className)} {...props}>
      {props.children}
    </div>
  );
};
```

_Note: please test this before merging._